### ISSUE TITLE
 Changing a device's position in a rack layout fails

### ISSUE THREAD

**Issue by karenetheridge** [joyent/conch-shell#196]
migrated from https://github.com/joyent/conch/issues/565

```
[daleg@iridium]~$ conch global l e40acbea-e9a7-4382-84df-57a14785526a update --ru-start 21
/: Properties not allowed: id.

```

**Comment by karenetheridge** [comment]
The json schema for 'RackLayoutUpdate' doesn't accept 'id' -- the layout id is taken from the path.

**Comment by sungo** [comment]
The bad data issue was resolved in #197
